### ISSUE TITLE
Most optimized settings for 4K on VP9?

### ISSUE THREAD

**Issue by davym123** [fnordware/AdobeWebM#20]
I know what you're thinking "This is not a support forum!". But seriously, I can't get a baseline of information about encoding in VP9, most information that I can find will be the code settings, never can I find a good setting for this plugin (which seem to use different terms or something?)

So I recently got into 4K filming and I exported my video in H.264. On my pc it looked great but when I uploaded to youtube it really compressed it and made it look bad. But other uploaders (like the 4K video from Youtube) seem to have super sharp images! But then I noticed that they were all using VP9, so then I began researched about it and it seems like it's the next big thing from Google.

Then I came across this plugin and I was very eager to try it. My video is 3 minutes long but when I start encoding the estimation of completion seems to be around 20 hours? I was wondering if this is normal? I know it's slow, but I never expected this, and my pc is not that bad. I used these settings: http://prntscr.com/6qm0nh

TL:DR - What are the best export settings to use to get a sharp video like this one (https://www.youtube.com/watch?v=wTcNtgA6gHs) but still being able to use your pc for 2 days?

Specs: 
Intel i7 2600
8GB RAM
Nvidia GTX 680

**Comment by fnordware** [comment]
I'm afraid that VP9 does take quite a while to encode. We've got some pretty decent default settings in there right now to try to get a reasonable speed from VP9 (make sure you're using a recent version), but there's only so much that can be done.

One thing is that 8 GB RAM is not a ton when you're encoding 4K. If you fill up your RAM you might start swapping, which will really slow things down.
